addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -107,6 +107,7 @@ $o-wslides-fs-side-width: 300px;
 
     .o_wslides_js_slide_like_up,
     .o_wslides_js_slide_like_down {
+        min-width: 2rem;
```

Comment:
One small bug I found is that if you spam click, it will select the text of the button (and it shouldn't imo)

You can use `class="user-select-none"` to fix that :) 

![screenshot_spect](https://user-images.githubusercontent.com/54316226/133231912-ae4932f1-11b9-48b2-81d8-c945e8cc975d.png)

Reply:
done!